omniledger/simulation/coins.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/BurntSushi/toml"
 	"github.com/dedis/cothority/omniledger/contracts"
 	"github.com/dedis/cothority/omniledger/darc"
-	"github.com/dedis/cothority/omniledger/service"
+	omniledger "github.com/dedis/cothority/omniledger/service"
```

Comment:
should we standardise on `ol` or `omniledger`? Now we have both...

Reply:
I vote for `ol` because it's shorter

Reply:
:+1: here.

Reply:
I see that I am outvoted, so ok. I like package names to be larger than 2 characters, because 2 characters are usually for local variables. See https://research.swtch.com/names